lib/to_openstudio/construction/air.rb

```diff
@@ -45,6 +45,9 @@ def find_existing_openstudio_object(openstudio_model)
     def to_openstudio(openstudio_model)
       os_construction = OpenStudio::Model::ConstructionAirBoundary.new(openstudio_model)
       os_construction.setName(@hash[:identifier])
+      unless @hash[:display_name].nil?
+        os_construction.additionalProperties().setFeature("Display Name", @hash[:display_name])
```

Comment:
@macumber This is added as a temporary fix till there is a new method. That makes sense, I can transfer this to the ModelObject, that way once there is a new method it would only need to be updated there.

Reply:
Cool.  Just to be clear, the model object method would still store data in the additionalProperties like you are now.  It would mostly just hard code the key name in C++

One other question, are you sure you want a space in the key name?  E.g. “DisplayName” vs “Display Name”?